src/Webflow/Api.php

```diff
@@ -204,6 +210,39 @@ public function findOrCreateItemByName(string $collectionId, array $fields)
         $this->cacheSet($cacheKey, $items);
         return $newItem;
     }
+   
+    public function createOrUpdateItemByName(string $collectionId, array $fields)
```

Comment:
Hi @jbardnz and thanks for the PR. Domains fix looks good, but `createOrUpdateItemByName` is a custom logic of your application. Could you please, take it out, so we can merge the rest.